pkg/types/baremetal/validation/platform.go

```diff
@@ -270,21 +270,44 @@ func validateOSImages(p *baremetal.Platform, fldPath *field.Path) field.ErrorLis
 	return platformErrs
 }
 
-func validateHostsCount(hosts []*baremetal.Host, installConfig *types.InstallConfig) error {
+func validateMastersCount(hosts []*baremetal.Host, installConfig *types.InstallConfig) error {
 
-	hostsNum := int64(len(hosts))
 	counter := int64(0)
+	if installConfig.ControlPlane != nil && installConfig.ControlPlane.Replicas != nil {
```

Comment:
If ControlPlane is nil or ControlPlane.Replicas is nil, is this still a valid config?

Reply:
For such cases there's a reasonable default [here](https://github.com/openshift/installer/blob/40373d2fbebcb79117b7fcf1bd92d6252630fdbe/pkg/types/defaults/installconfig.go#L68-L72)

Reply:
Note that the default is applied prior to validation.